org.envirocar.app/res/layout/activity_signin.xml

```diff
@@ -66,7 +66,7 @@
             android:id="@+id/activity_signin_username_input_layout"
             style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
             android:layout_width="0dp"
-            android:layout_height="40dp"
+            android:layout_height="wrap_content"
```

Comment:
the layout_height is changed to ``wrap_content`` because the layout needs to be big enough, to show the error.
more information: https://stackoverflow.com/a/44831214 
see the below comments on this answer :)